Attributed/UIFont.swift

```diff
@@ -32,33 +32,48 @@ extension UIFont {
     }
     
     func fontWithMonospacedNumbers() -> UIFont {
-        return UIFont(descriptor: fontDescriptor.addingAttributes([
-            UIFontDescriptorFeatureSettingsAttribute: [
-                [
-                    UIFontFeatureTypeIdentifierKey: kNumberSpacingType,
-                    UIFontFeatureSelectorIdentifierKey: kMonospacedNumbersSelector,
-                ],
-            ],
-            ]), size: pointSize)
+        #if os(watchOS)
```

Comment:
Yeah, this fails building for watchOS:
 -> Attributed (3.0.0)
    - ERROR | [watchOS] xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - ERROR | [watchOS] xcodebuild:  /Users/brandenrussell/Code/Attributed/Attributed/UIFont.swift:41:57: error: use of unresolved identifier 'kNumberSpacingType'
    - ERROR | [watchOS] xcodebuild:  /Users/brandenrussell/Code/Attributed/Attributed/UIFont.swift:42:61: error: use of unresolved identifier 'kMonospacedNumbersSelector'